tests/bsim/bluetooth/audio_samples/compile.sh

```diff
@@ -18,6 +18,12 @@ if [ "${BOARD_TS}" == "nrf5340bsim_nrf5340_cpuapp" ]; then
   app=tests/bsim/bluetooth/audio_samples/broadcast_audio_sink sysbuild=1 \
     conf_file=${ZEPHYR_BASE}/samples/bluetooth/broadcast_audio_sink/prj.conf \
     exe_name=bs_${BOARD_TS}_${app}_prj_conf sysbuild=1 compile
+  app=tests/bsim/bluetooth/audio_samples/cap_initiator sysbuild=1 \
+    conf_file=${ZEPHYR_BASE}/samples/bluetooth/broadcast_audio_sink/prj.conf \
```

Comment:
is this a copy paste error or you really want different conf files for the nrf52 and 53?

Reply:
Spelling mistake :)